postmark.php

```diff
@@ -196,6 +246,91 @@ function settings_html() {
     }
 }
 
+// Creates a table for storing logs of send attempts.
+function pm_log_create_db() {
+
+  global $wpdb;
+
+  $table_name = $wpdb->prefix . 'postmark_log';
+
+  // Creates a logs table if it doesn't exist already.
+  if( $wpdb->get_var("SHOW TABLES LIKE '$table_name'") != $table_name ) {
+
+    $charset_collate = $wpdb->get_charset_collate();
+
+    $sql = "CREATE TABLE $table_name (
```

Comment:
This table should have an index on `time` -- I would also recommend calling the field something less keyword-y, like `log_entry_date`.

Stick with using normal INT for the id.

Reply:
Switched column name for time to log_entry_date, added index on it, and switched to regular ole INT for the id.